client/lib/plans/features-list.js

```diff
@@ -784,15 +784,15 @@ export const FEATURES_LIST = {
 	},
 	[ constants.FEATURE_ACTIVITY_LOG ]: {
 		getSlug: () => constants.FEATURE_ACTIVITY_LOG,
-		getTitle: () => i18n.translate( 'Expanded Site Activity' ),
+		getTitle: () => i18n.translate( 'Expanded site activity' ),
 		getDescription: () =>
 			i18n.translate(
 				'Take the guesswork out of site management and debugging with a filterable record of all the activity happening on your site.'
 			),
 	},
 	[ constants.FEATURE_POLLS_PRO ]: {
 		getSlug: () => constants.FEATURE_POLLS_PRO,
-		getTitle: () => i18n.translate( 'Advanced Polls and Ratings' ),
+		getTitle: () => i18n.translate( 'Advanced polls and ratings' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.